### PR TITLE
Improve setup appliance

### PR DESCRIPTION
**Pull request by M0ses**
* change system-user-* to new mechanism
* Redirect error message in setup-appliance.sh db:migrate as it`s only used for testing and may fail on first startup
* Do not run setup-appliance.sh parallel
*

### CODE REVIEW

File: dist/setup-appliance.sh [hunk 1]
```diff
@@ -709,9 +710,23 @@ fi
 
 
 if [[ ! $BOOTSTRAP_TEST_MODE == 1 && $0 != "-bash" ]];then
-
+  logline "Starting "`basename $0`" at "`date`
   NON_INTERACTIVE=0
 
+  if [ -f $PID_FILE ];then
+    APID=`cat $PID_FILE`
+    if [ ! -f /proc/$APID/status ];then
```
Comment (hennevogel) [inline dist/setup-appliance.sh:718]: If you switch the conditions around it's way easier to understand IMHO

Comment (M0ses) [reply dist/setup-appliance.sh:718]: I didn't get the point - please explain!

Comment (hennevogel) [reply dist/setup-appliance.sh:718]:
```bash
if true; then
...
else
...
fi
```

vs.

```bash
if !true; then
...
else
...
fi
```

Comment (M0ses) [reply dist/setup-appliance.sh:718]: Feel free to send an PR - and test it - if you really unhappy with the current solution.
Testing these changes is a lot of effort and I'm not willing to do this effort only for "coding style"

Comment (eduardoj) [reply dist/setup-appliance.sh:718]: I'm with Henne here. I will create a PR later, with his suggestion, so we can also merge this PR.

**Review by eduardoj — APPROVED** [review]

**Pull request merged**